src/redux/mapStoreToProps.ts

```diff
@@ -9,10 +9,21 @@ export interface Auth {
   email: string,
   token: string,
   user: {
-    userType?: string;
+    userType: string;
```

Comment:
@Unesdala userType is not required, here is the backend model:
https://github.com/WebJamApps/web-jam-back/blob/dev/model/user/user-schema.js